mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSourceSchemaReader.java

```diff
@@ -34,6 +34,17 @@ public class SqlServerSourceSchemaReader extends CommonSchemaReader {
   public static final int SQL_VARIANT = -156;
   public static final String DATETIME_TYPE_PREFIX = "datetime";
 
+  private String sessionID;
+
+  public SqlServerSourceSchemaReader() {
+    super();
```

Comment:
similar comment, call the other constructor

Reply:
Fixed!